compiler-rt/lib/mpk_untrusted/mpk_fault_handler.cpp

```diff
@@ -0,0 +1,98 @@
+#include "mpk_fault_handler.h"
+#include "alloc_site_handler.h"
+#include "mpk.h"
+
+#include <sys/mman.h>
+
+// Set to whatever the default page size will be for page based MPK.
+#define PAGE_SIZE 4096
```

Comment:
This is probably safe(especially since I also do this in places), but we can get this number at runtime w/ `getpagesize()`